docs/references/users/create-user-jwt.md

```diff
@@ -0,0 +1 @@
+Use this endpoint to create a JSON Web Token for user by its unique ID. You can use the resulting JWT to authenticate on behalf of the user. The JWT secret will become invalid if the session it uses gets deleted.
```

Comment:
> The JWT secret will become invalid if the session it uses gets deleted.

We actually have a bug with this: 
* https://github.com/appwrite/appwrite/issues/8000

But hopefully it will be fixed soon.

Reply:
Good to know. This is going to 1.6.x, We have enough time to have that fix implement before 1.6.x